postile/postile.py

```diff
@@ -117,6 +119,14 @@ def prepared_query(filename):
         headers={"Content-Type": "application/json"}
     )
 
+@app.route('/fonts/<fontstack:string>/<frange:string>.pbf')
+async def get_fonts(request, fontstack, frange):
+    if not Config.fonts:
+        return response.text('no fonts available', status=404)
+    return await response.file(
+        Path(Config.fonts) / fontstack / f'{frange}.pbf',
```

Comment:
Do we use f-strings already? Do we care about Python < 3.6?

Reply:
Yes, f-strings are already used in code, and Python3.6 is mentioned as a requirement in the readme.